### ISSUE TITLE
Visible focus

### ISSUE THREAD

**Issue by davidbenque** [cryptpad/cryptpad#1206]
> Most of the time the focus is not visible. For keyboard users who are not blind it’s hard or even impossible to see where they are on the page. 

We should make a deliberate design decision about how focus looks (rounded corners, brand color?) and apply consistently everywhere
